test/test-project/gulpfile.js

```diff
@@ -48,5 +68,21 @@ gulp.task('test1', () => {
     // TODO(justinfagnani): allow filtering of files before writing
     .pipe(gulp.dest('build/unbundled'));
 
-  return mergeStream(bundled, unbundled);
+  return Promise.all([waitFor(bundled), waitFor(unbundled)]).then(() => {
+    return Promise.all([
+      addServiceWorker({
```

Comment:
Actually, if we do the work of waiting for streams to finish in `addServiceWorker` we could do this:

```javascript
return Promise.all([
  addServiceWorker(project.unbundledConfig, unbundled),
  addServiceWorker(project.bundledConfig, bundled),
]);
```